auxiliaries.py

```diff
@@ -214,7 +215,8 @@ def __init__(self,
         self.down_interval_s = down_interval_s
         self.led = network_led
         self.blink_period_s = 1.5
-
+        self.last_up_time = None
+        
```

Comment:
Let's call it `last_try_time` since it gets updated even when the network is down (when the ifdown/ifup is run).